hack/observability/opentelemetry/controller-manager-patch.yaml

```diff
@@ -14,3 +14,4 @@ spec:
             - "--leader-elect"
             - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=true},AKSResourceHealth=${EXP_AKS_RESOURCE_HEALTH:=false},EdgeZone=${EXP_EDGEZONE:=false},ASOAPI=${EXP_ASO_API:=true}"
             - "--enable-tracing"
+            - "--v=4"
```

Comment:
Increases logging across CAPZ Controller manager. 
So `log.V(2).Info("haha")` while local dev works.

@willie-yao ☝🏼

Reply:
we can remove this from the PR now